coretex/cli/modules/config_defaults.py

```diff
@@ -6,14 +6,16 @@
 from ...statistics import getAvailableRamMemory
 
 
+cpuCount = os.cpu_count()
+
 DOCKER_CONTAINER_NAME = "coretex_node"
 DOCKER_CONTAINER_NETWORK = "coretex_node"
 DEFAULT_STORAGE_PATH = str(Path.home() / ".coretex")
 DEFAULT_RAM_MEMORY = getAvailableRamMemory()
 MINIMUM_RAM_MEMORY = 6
 DEFAULT_SWAP_MEMORY = DEFAULT_RAM_MEMORY * 2
 DEFAULT_SHARED_MEMORY = 2
-DEFAULT_CPU_COUNT = os.cpu_count()
+DEFAULT_CPU_COUNT: int = cpuCount if cpuCount is not None else 0
```

Comment:
Create a function for this in `statistics` module. If `os.cpu_count()` returns `None` return 1 not 0.